doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -324,10 +418,23 @@ Insufficiently restrictive variance annotations lead to errors:
 
     Inductive eq@{i} {A : Type@{i}} (x : A) : A -> Type@{i} := eq_refl : eq x x.
 
-    Definition funext_type@{a b e} (A : Type@{a}) (B : A -> Type@{b})
-    := forall f g : (forall a, B a),
-                    (forall x, eq@{e} (f x) (g x))
-                    -> eq@{e} f g.
+.. coqtop:: none
+
+   Set Printing Universes.
+
+.. coqtop:: out
+
+   Print eq.
+
+The universe of :g:`eq` is irrelevant here, hence proofs of equalities can
+inhabit any universe.  The universe must be big enough to fit `A`.
+
+.. coqtop:: in
+
+   Definition funext_type@{a b e} (A : Type@{a}) (B : A -> Type@{b})
+   := forall f g : (forall a, B a),
+                   (forall x, eq@{e} (f x) (g x))
+                   -> eq@{e} f g.
```

Comment:
There's an indentation problem that makes this example "escape" the example box.